homeassistant/components/pegel_online/sensor.py

```diff
@@ -36,24 +36,87 @@ class PegelOnlineSensorEntityDescription(
 
 
 SENSORS: tuple[PegelOnlineSensorEntityDescription, ...] = (
+    PegelOnlineSensorEntityDescription(
+        key="air_temperature",
+        translation_key="air_temperature",
+        measurement_key="air_temperature",
+        state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        icon="mdi:thermometer-lines",
```

Comment:
I suppose we should remove the icons where we have a device_class to use the ones associated with that device class

Reply:
Otherwise I think this would be done

Reply:
maybe for the ph entity, but all others has intentionally other icons to differentiate them (_eq. air vs. water temperature_) or just to be more meaningful (_eg. the water flow speed or clearance height_)